sentry/src/test/java/io/sentry/SentryEnvelopeTest.kt

```diff
@@ -137,4 +135,25 @@ abcdefghij""".toInputStream()
         assertEquals(10, secondItem.header.length)
         assertEquals(10, secondItem.data.size)
     }
+
+    @Test
+    fun `deserializes an user feedback`() {
+        val envelope = readEnvelope("envelope-feedback.txt")
```

Comment:
`l`: I think we could rename the `envelope-feedback.txt` to `envelope-feedback-from-dart.json` to reflect it is an envelope from dart and it contains json.

Reply:
the same issue would happen with an envelope from react-native, or sentry-native, so it's rather a valid feedback envelope, I'd keep as it is

Reply:
Fine with me. 🚢